minecraft_servers/aspaku/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+  "server_name": "aspaku",
+  "nice_name": "Aspaku",
+  "direct_ip": "mc.aspaku.com",
+  "server_wildcards": [
+    "%.aspaku.com"
+  ],
+  "social": {
+    "web": "https://aspaku.com/",
```

Comment:
Please remove the trailing slash on the website url.

Reply:
i'll do that, sorry aha ^^

Reply:
good !